io.sourceforge.chart-geany.chart-geany.yml

```diff
@@ -0,0 +1,57 @@
+app-id: io.sourceforge.chart_geany.chart_geany
```

Comment:
manifest file name MUST match this. Also I think the app name should be `chart-geany`. `-` is allowed in the app name

Reply:
changed